components/layout/block.rs

```diff
@@ -1787,6 +1787,17 @@ impl BlockFlow {
         self.flags.contains(HAS_SCROLLING_OVERFLOW)
     }
 
+    // Return offset from original position because of `position: sticky`.
+    pub fn sticky_position(&self) -> (MaybeAuto, MaybeAuto, MaybeAuto, MaybeAuto) {
```

Comment:
Would the return type be better as a `LogicalMargin`?

Reply:
Sure. I can use `LogicalMargin` here and return a `SideOffsets2D` after converting it to physical dimension.